package.json

```diff
@@ -8,35 +8,32 @@
   "license": "LGPL-3.0-or-later",
   "scripts": {
     "build": "tsc",
-    "test": "mocha build/test/**/*.test.js",
+    "test": "yarn jest",
     "tsc": "tsc",
     "clean": "rimraf build/"
   },
   "devDependencies": {
-    "@types/chai": "^4.1.3",
-    "@types/chai-as-promised": "^7.1.0",
     "@types/debug": "^0.0.30",
-    "@types/mocha": "^5.2.1",
+    "@types/jest": "^23.3.12",
     "@types/mock-fs": "^3.6.30",
     "@types/nock": "^9.1.3",
     "@types/node": "^9.6.5",
     "@types/request": "^2.47.0",
     "@types/tmp": "^0.0.33",
-    "chai": "^4.1.2",
-    "chai-as-promised": "^7.1.1",
-    "mocha": "^5.2.0",
-    "mock-fs": "^4.5.0",
+    "jest": "^23.6.0",
     "nock": "^9.3.2",
     "rimraf": "^2.6.2",
+    "ts-jest": "^23.10.5",
     "typescript": "^2.8.1"
   },
   "dependencies": {
     "debug": "^3.1.0",
     "hosted-git-info": "^2.6.0",
+    "memfs": "^2.14.2",
     "request": "^2.85.0",
     "tmp": "^0.0.33"
   },
   "engines": {
-    "node": "<10.5"
+    "node": ">=10.0.0"
```

Comment:
The tests probably work on Node 9. Remove the `engines` object altogether

Reply:
Yeah, but I'd really like to dissuade anyone from using those awkward, in-between-LTS versions of node.

Reply:
@cymerrad That's not up to us to decide, engine definitions might disable using our package altogether even if it works fine - http://www.marcusoft.net/2015/03/packagejson-and-engines-and-enginestrict.html